include/json/reader.h

```diff
@@ -271,6 +271,12 @@ class JSON_API CharReader {
      * \throw std::exception if something goes wrong (e.g. invalid settings)
      */
     virtual CharReader* newCharReader() const = 0;
+
+    /** \brief Allocate a CharReader via newCharReader(). 
+     *   wrap the object in std::unique_ptr<CharReader> to esnure deletion.
```

Comment:
spelling here.

But you don't have to say why it's being wrapped in unique_ptr. Everybody knows what unique_ptr is.

    /** Wrap newCharReader's result in a std::unique_ptr. Throws if newCharReader throws. */

And I think that's it. They'll look up the rest of the behavior from `newCharReader`.